projects/ccd-case-ui-toolkit/tsconfig.lib.json

```diff
@@ -13,7 +13,7 @@
     "types": [],
     "lib": [
       "dom",
-      "es2018"
+      "es2019"
```

Comment:
@MateiHMCTS - any reason why this is changed to 2019?

Reply:
I wanted to use Array.prototype.flat()